robot/basestation/stream_capture.py

```diff
@@ -91,25 +151,30 @@ def start_ffmpeg_record(stream_url, filename):
     """
     proc_video[stream_url] = subprocess.Popen(['ffmpeg -i ' + stream_url + ' -acodec copy -vcodec copy ' + filename], stdin=PIPE, shell=True)
 
-def stream_capture(stream_url):
-    """ Given a stream, captures an image.
+def stream_capture(stream_url, camera_rotation):
+    """ Given a stream, captures an image and rotates it as shown in GUI
 
         stream_url : The URL of the stream to capture the image.
+        camera_rotation : The rotation value (0,1,2,3) of the stream as shown
+        in the GUI. 0 = no rotation. 1 = clockwise 90 degrees
+        2 = 180 degrees. 3 = counterclockwise 90 degrees
```

Comment:
You might want to format it just like you did for the other function (by making it multiple lines) for the sake of consistency